rwengine/src/render/TextRenderer.cpp

```diff
@@ -12,18 +12,22 @@
 #include "engine/GameData.hpp"
 #include "render/GameRenderer.hpp"
 
-int charToIndex(uint16_t g) {
+unsigned charToIndex(std::uint16_t g) {
     // Correct for the default font maps
     /// @todo confirm for JA / RU font maps
     return g - 32;
 }
 
-glm::vec4 indexToCoord(font_t font, int index) {
-    float x = static_cast<int>(index % 16);
-    float y = static_cast<int>(index / 16) + 0.01f;
-    float fontHeight = ((font == FONT_PAGER) ? 16.f : 13.f);
-    glm::vec2 gsize(1.f / 16.f, 1.f / fontHeight);
-    return glm::vec4(x, y, x + 1, y + 0.98f) * glm::vec4(gsize, gsize);
+static glm::vec4 indexToTexCoord(int index, const glm::u32vec2 &textureSize, const glm::u8vec2 &glyphOffset) {
```

Comment:
anonymous namespace?

Reply:
Just to avoid the `static` keyword?

Reply:
Not sure if `static` is even needed.
Btw: http://www.comeaucomputing.com/techtalk/#nostatic

Reply:
``charToIndex``, ``indexToCoord``, the shaders, and the font widths can all be ``static`` or in an anonymous namespace. Wrapping them all in a namespace is the expected C++ way.